rllib/core/learner/torch/torch_learner.py

```diff
@@ -257,15 +262,57 @@ def apply_gradients(self, gradients_dict: ParamDict) -> None:
                         "`False`."
                     )
 
-                    # If the module uses learning rate schedulers, step them here.
-                    if module_id in self._lr_schedulers:
-                        for scheduler in self._lr_schedulers[module_id][optimizer_name]:
-                            scheduler.step()
-
-                    # If the module uses learning rate schedulers, step them here.
-                    if module_id in self._lr_schedulers:
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
```

Comment:
Great! This is so much better to do this in this method here (instead of in `apply_gradients`).